parts/openshift/unstable/openshiftmasterscript.sh

```diff
@@ -95,14 +97,24 @@ for i in /etc/origin/master/master-config.yaml /tmp/bootstrapconfigs/* /tmp/ansi
     sed -i "s|HOSTNAME|${HOSTNAME}|g;" $i
 done
 
+# backup the current node.kubeconfig
+cp /etc/origin/node/bootstrap.kubeconfig /etc/origin/node/bootstrap.kubeconfig.orig
```

Comment:
why is this done?

Reply:
I think the desire was to replace the admin.kubeconfig with the original kubeconfig file so that the node doesn't use the admin.kubeconfig.

On line 149, I move it back into place so that it is the original.

I think we need a strategy to use a non-privileged kubeconfig.